gnoi/os/client.go

```diff
@@ -28,11 +32,139 @@ type Client struct {
 	client pb.OSClient
 }
 
+const chunkSize = 5000000
+
 // NewClient returns a new OS service client.
 func NewClient(c *grpc.ClientConn) *Client {
 	return &Client{client: pb.NewOSClient(c)}
 }
 
+// Install invokes the Install RPC for the OS service.
+func (c *Client) Install(ctx context.Context, imgPath, version string, printStatus bool, validateTimeout time.Duration) error {
+	// Read file to buffer.
+	file, err := ioutil.ReadFile(imgPath)
```

Comment:
I know this complicates things, but imagine the OS file is 1G (normal for some platforms). Would it be better to load chunks of it instead? =)

Reply:
Oh i get you. Should be done :)